xml/poc-prerequisites.xml

```diff
@@ -0,0 +1,146 @@
+ title="Profiling step"?>
+<!DOCTYPE chapter [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="poc-prerequisites">
+ <title>Before you begin</title>
+ <section xml:id="hardware-requirements">
+  <title>Minimum hardware requirements</title>
+  <para>
+    Before you begin, the following minimum hardware requirements are needed:
+  </para>
+    <itemizedlist>
+      <listitem>
+        <para>
+          1 Deployer node
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          2 Compute nodes
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          3 Controller nodes
+        </para>
+      </listitem>
+    </itemizedlist>
+ </section>
+
+ <section xml:id="software-requirements">
+  <title>Minimum software requirements</title>
+  <para>The following minimum software requirements are needed:
+  </para>
+    <itemizedlist>
+      <listitem>
+        <para>
+          KVM hypervisor
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <emphasis>OpenStack services:</emphasis> Identity service (keystone), Compute service (nova),
+          Block Storage service (cinder), Image service (glance),
+          Networking service (neutron), Dashboard service (horizon)
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          &ses; powered by &ceph;
+        </para>
+      </listitem>
+     </itemizedlist>
+  </section>
+
+ <section xml:id="configuration">
+  <title>Configuration</title>
+  <para>
+    The following configuration tools are required for a POC deployment:
+  </para>
+  <para>
+    **Subscription Management Tool (SMT)**: Used for SUSE Linux Enterprise 12 SP4
+    to allow enterprise customers to optimize the management of SUSE Linux
+    Enterprise software updates and subscription entitlements. SMT allows
+    you to provision updates for all of your devices running a product
+    based on SUSE Linux Enterprise.
+  </para>
+  <para>
+    **SUSE Customer Center (SCC)**: Before you create a local mirror of the
+    required repositories, you need appropriate organization credentials. You
+    can obtain the credentials from SCC. To get the credentials from the SCC, visit
+    the SUSE Customer Center at http://scc.suse.com and log in.
+  </para>
+  <para>
+    Mirror the following repositories:
+    <itemizedlist>
+      <listitem>
+        <para>
+        SLES12-SP4-Pool x86_64
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+        SLES12-SP4-Updates x86_64
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+        SUSE-OpenStack-Cloud-9-Pool x86_64
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+        SUSE-OpenStack-Cloud-9-Updates x86_64
+        </para>
+      </listitem>
+    </itemizedlist>
+  </para>
+  <para>
+    NFS
```

Comment:
its not clear what this NFS represents

Reply:
I admittedly have assumed knowledge in this guide of at least system admin or above, and in that case, they should be aware that NFS is Network File System. I'll add it in if you think that might still be tricky?